src/js/InterfaceManager.js

```diff
@@ -0,0 +1,232 @@
+import { NavLink } from "react-router-dom";
+/**
+ * Get an array of prepared tracks to insert into HTML.
+ * @param  {} array - Array with tracks formed by spotify.js
```

Comment:
Неправильно написано. Должно быть как-то так:
```suggestion
 * @param  {array} array - Array with tracks formed by spotify.js
```

Это относится ко всем местам

Reply:
Поправил